etc/profile-a-l/akregator.profile

```diff
@@ -19,9 +20,11 @@ include disable-shell.inc
 
 mkfile ${HOME}/.config/akregatorrc
 mkdir ${HOME}/.local/share/akregator
+mkdir ${HOME}/.local/share/kxmlgui5/akregator
```

Comment:
If `${HOME}/.local/share/kxmlgui5/akregator` is a file, this should be `mkfile`.

Reply:
It's a directory on my system.